samples/SPLToJsonConvert/info.xml

```diff
@@ -5,12 +5,12 @@
    <info:name>com.ibm.streamsx.json.SPLToJsonConvert</info:name>
    <info:description></info:description>
    <info:version>1.0.0</info:version>
-   <info:requiredProductVersion>4.0.1.0</info:requiredProductVersion>
+   <info:requiredProductVersion>4.2.0.0</info:requiredProductVersion>
```

Comment:
The functions are new to the toolkit not the IBM Streams product, the sample will work against older releases when using the new version of the JSON toolkit.